engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -117,48 +115,14 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
   private val tag = self.path.name
 
   private var abortingWorkflowToReplyTo = Map.empty[WorkflowId, ActorRef]
-  private var shutdownHookThreadOption: Option[ShutdownHookThread] = None
+  private var nextPollCancellable: Option[Cancellable] = None
+  private var preventNewWorkflowsFromStarting: Boolean = false
```

Comment:
I think I found a way to do it all with states - it's arguably cleaner but the `var` is gone. I'll push the change once I've finished addressing the rest of the comments